Line chart without the y-axis

```diff
@@ -0,0 +1,95 @@
+CustomChart {
```

Comment:
Please create a sub directory for your files, e.g:
- `line_chart_without_yaxis/line_chart_without_yaxis.vgl.aml`
- `line_chart_without_yaxis/README.md`